airflow/contrib/operators/kubernetes_operator.py

```diff
@@ -96,30 +102,31 @@ def poll_job_completion(self):
     def execute(self, context):
         # create job_yaml_string from the parameter dictionary.
         # append extra parameters to commands inside the job
-        if not self.job_yaml_string and self.job_param_dict and 'containers' in self.job_param_dict:
+        instance_param_dict = self.job_param_dict.copy()
+        instance_param_dict['context'] = context
+        instance_param_dict['config'] = configuration.as_dict()
```

Comment:
I see we want to get enable_xcom_pickling out of config.  So this is set up per whole airflow instance.  We don't allow users to choose pickling or not here?